_posts/2021-10-04-insertion-sort.md

```diff
@@ -0,0 +1,123 @@
+---- 
+  title:  "Insertion Sort"
+  description:  "Insertion sort is a simple sorting technique. Here in this post, first we will see the algorithm, then we will see an example of its working, we will also have a look on its implementation in C++ language, then in the last we will discuss its other characteristics."
+  category: dsa
+  image: /assets/images/categories/algorithm.jpg
+  keywords: # Keywords your want to ad for SEO
+    - Mentor 
+    - Mentro
+    - Mentorship
+  author: TD-17
+  permalink: /dsa/:title/
+  tags: 
+    - sorting algorithm
+    - dsa
+----
+
+## Insertion Sort

```

Comment:
leave one line after the sub-headings i.e leave one line after ## Insertion Sort heading and follow it for every sub-heading